WalletWasabi.Fluent/ViewModels/AddWallet/WelcomePageViewModel.cs

```diff
@@ -0,0 +1,20 @@
+using System.Reactive;
+using System.Windows.Input;
+using ReactiveUI;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+
+namespace WalletWasabi.Fluent.ViewModels.AddWallet
+{
+	public partial class WelcomePageViewModel : DialogViewModelBase<Unit>
+	{
+		public WelcomePageViewModel()
+		{
+			GetStartedCommand = ReactiveCommand.Create(() => Close());
```

Comment:
no, because we want the background to stay until the end of the wallet creation when its a completely new setup.

Reply:
It will because the background goes hidden only if Oobe = false and the wallet list is not empty.

```
if (Services.WalletManager.HasWallet() && !Services.UiConfig.Oobe)
{
	IsSetup = true;
}
```